test/components/previews/daisy_ui/data_display/alert_preview/alert_dash_style.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= render DaisyUI::Alert.new(text: '12 unread messages. Tap to see.', color: :info, variant: :dash, icon: false) %>
+<%= render DaisyUI::Alert.new(text: 'Your purchase has been confirmed!', color: :success, variant: :dash, icon: false) %>
+<%= render DaisyUI::Alert.new(text: 'Warning: Invalid email address!', color: :warning, variant: :dash, icon: false) %>
+<%= render DaisyUI::Alert.new(text: 'Error! Task failed successfully.', color: :error, variant: :dash, icon: false) %>
```

Comment:
THere should not be the need to pass icon:false 
If no icon is pass not icon is shown